### PR TITLE
[EdgeDB] OtherProduct seeds

### PR DESCRIPTION
**Pull request by willdch**
https://seed-company-squad.monday.com/boards/5989610236/pulses/5793006654

### CODE REVIEW

**Review by bryanjnelson — CHANGES_REQUESTED** [review]
Looks great @willdch!  Much better formatting as well!  😉

File: dbschema/seeds/014.product.edgeql [hunk 1]
```diff
@@ -1083,3 +1083,121 @@ with
...
+          methodology := <Product::Methodology>json_get(otherProduct,'methodology'),
+          describeCompletion := <str>json_get(otherProduct,'describeCompletion'),
+          progressTarget := <int16>otherProduct['progressTarget'],
+          progressStepMeasurement := <str>otherProduct['progressStepMeasurement']
+        })
+      )
+    )
+  ),
+  newOtherProducts := (select otherProducts filter .createdAt = datetime_of_statement())
+  select { `Added Other Products` := newOtherProducts.project.name ++ ': ' ++ newOtherProducts.title }
```
Comment (bryanjnelson) [inline dbschema/seeds/014.product.edgeql:1203]: Remove the indention on the last line here.  It should be flush on the left.  Also, we always want one new line at the end of every file.  👊🏼

File: dbschema/seeds/014.product.edgeql [hunk 2]
```diff
@@ -1090,20 +1090,15 @@ with
...
     "progressStepMeasurement": "Percent"
   },
   {
     "project": "Misty Mountains",
     "title":  "7 new songs written and recorded",
-    "mediums": [
-      "Audio",
-      "Web"
-      ],
+    "mediums": ["Audio","Web"],
```
Comment (bryanjnelson) [inline dbschema/seeds/014.product.edgeql:1101]: We want spaces in between our array entries, so like this here...and then on any others.

```suggestion
    "mediums": ["Audio", "Web"],
```

**Review by bryanjnelson — CHANGES_REQUESTED** [review]

**Review by bryanjnelson — APPROVED** [review]
Thanks @willdch!  LGTM!

@CarsonF?

**Comment by CarsonF** [comment]
Let's put these in their own files. I don't want to try to find the actual logic here in the middle of thousands of lines.
Not to mention your two PRs conflict with each other since they are both modifying the same spot.

**Review by CarsonF — APPROVED** [review]

**Pull request merged**